src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+    protected String start;
+    protected String end;

```

Comment:
Any reason why you chose to set the access modifier of these two variables to `protected` instead of `private`?